src/abyssal_tome/constants.py

```
@@ -0,0 +1,99 @@
+CYCLES_MAP: dict[str, str] = {
+    '01': 'core', '02': 'dwl', '03': 'ptc', '04': 'tfa', '05': 'tcu',
+    '06': 'tde', '07': 'tic', '08': 'eoe', '09': 'tsk', '50': 'rtnotz',
+    '51': 'rtdwl', '52': 'rtptc', '53': 'rttfa', '54': 'rttcu',
+    '60': 'investigator', '61': 'investigator', '62': 'investigator',
+    '63': 'investigator', '64': 'investigator', '81': 'standalone',
+    '82': 'standalone', '83': 'standalone', '84': 'standalone',
+    '85': 'standalone', '86': 'standalone', '90': 'parallel', '98': 'books', '99': 'promo',
+}
+
+# --- Constants for scripts/process_json.py ---
+
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)